src/web/services/deadline-extension-helper.ts

```diff
@@ -97,4 +97,17 @@ export class DeadlineExtensionHelper {
     });
     return record;
   }
+
+  public static getUserFeedbackSessionEndingTimestamp(feedbackSession: FeedbackSession): number {
+    if (DeadlineExtensionHelper.hasUserOngoingExtension(feedbackSession)) {
+      return feedbackSession.submissionEndWithExtensionTimestamp!;
+    }
+    return feedbackSession.submissionEndTimestamp;
+  }
```

Comment:
Just wondering something. Does this mean that once the extension is over, the displayed end time will revert back to the usual unextended end time?

Reply:
@FergusMok just checking, has this been resolved?

Reply:
Thanks for the catch, I've updated for this case and in the screenshots too